src/applications/vaos/new-appointment/components/CommunityCareProviderSelectionPage/ProviderSelectionField.jsx

```diff
@@ -1,5 +1,236 @@
+            <LoadingIndicator message="Loading the list of providers" />
+          </div>
+        )}
+      {requestStatus === FETCH_STATUS.failed &&
+        showProvidersList && (
+          <div className="vads-u-padding-bottom--2">
+            <ErrorMessage />
+          </div>
+        )}
+      {requestStatus === FETCH_STATUS.succeeded &&
+        showProvidersList && (
+          <>
+            <h2 className="vads-u-font-size--h3 vads-u-margin-top--0">
+              Choose a provider
+            </h2>
+            <p>Your address on file:</p>
+            <ResidentialAddress address={address} />
+            <p>
+              Displaying 1 to {currentlyShownProvidersList.length} of{' '}
+              {communityCareProviderList.length} providers
+            </p>
+            {currentlyShownProvidersList.map(provider => {
+              const { name, position } = provider;
+              const checked = provider.id === checkedProvider;
+              const distance = distanceBetween(
+                position.latitude,
+                position.longitude,
+                address.latitude,
+                address.longitude,
+              );
+              return (
+                <div className="form-radio-buttons" key={provider.id}>
+                  <input
+                    type="radio"
+                    checked={checked}
+                    id={`${idSchema.$id}_${provider.id}`}
+                    name={`${idSchema.$id}`}
+                    value={provider.id}
+                    onChange={_ => setCheckedProvider(provider.id)}
+                    disabled={loadingProviders}
+                  />
+                  <label htmlFor={`${idSchema.$id}_${provider.id}`}>
+                    <span className="vads-u-display--block vads-u-font-weight--bold">
+                      {name}
+                    </span>
+                    <span className="vads-u-display--block">
+                      {provider.address?.line}
+                    </span>
+                    <span className="vads-u-display--block">
+                      {provider.address.city}, {provider.address.state}{' '}
+                      {provider.address.postalCode}
+                    </span>
+                    <span className="vads-u-display--block vads-u-font-size--sm vads-u-font-weight--bold">
+                      {distance} miles
+                    </span>
+                  </label>
+                  {checked && (
+                    <button
+                      type="button"
+                      onClick={() => {
+                        onChange(
+                          communityCareProviderList.find(
+                            p => p.id === checkedProvider,
+                          ),
+                        );
+                        setCheckedProvider();
+                        setShowProvidersList(false);
+                      }}
+                    >
+                      Choose Provider
```

Comment:
Should be sentence case ("Choose provider")

Reply:
Fixed